vendor/engines/secure_rooms/app/services/secure_rooms/check_access.rb

```diff
@@ -0,0 +1,24 @@
+module SecureRooms
+
+  class CheckAccess
+
+    DEFAULT_RULES = [
+      AccessRules::OperatorRule,
+      AccessRules::AccountSelectionRule,
+      AccessRules::DefaultRestrictionRule,
```

Comment:
Is there a strong reason to use the class method `call` rather than an instance method, and load a `new` as we loop over them?